app/models/application_form.rb

```diff
@@ -227,9 +227,7 @@ def english_language_exempt?
   end
 
   def secondary_education_teaching_qualification_required?
-    CountryCode.secondary_education_teaching_qualification_required?(
-      country.code,
-    )
+    country&.subject_limited || false
```

Comment:
```suggestion
    country.subject_limited
```

`country` won't be `nil` here so we don't need to access it safely.